docs/README.md

```diff
@@ -9,7 +9,7 @@ A JavaScript application framework for machine learning and its engineering.
 
 ## Why Pipcook
 
-With the mission of enabling JavaScript engineers to utilize the power of machine learning without any prerequisites and the vision to lead front-end technical field to the intelligention. [Pipcook][] is to become the JavaScript application framework for the cross-cutting area of machine learning and front-end interaction.
+With the mission of enabling JavaScript engineers to utilize the power of machine learning without any prerequisites and the vision to lead front-end technical field to the intelligence. [Pipcook][] is to become the JavaScript application framework for the cross-cutting area of machine learning and front-end interaction.
```

Comment:
This seems not related.

Reply:
Typos are disturbing :(

Reply:
May I ask you to open another PR for this?

Reply:
Yeah, of cause~